src/api/session.ts

```diff
@@ -88,9 +88,9 @@ function isLoggedIn(settings: ApiSettings) {
   return !!settings.user;
 }
 
-async function logOut(setUser: UserUpdater) {
+async function logOut(setUser: UserUpdater, skipRedirect = false) {
   setUser(null);
-  window.location.href = urls.logOut;
+  skipRedirect && (window.location.href = urls.logOut);
```

Comment:
I'd prefer an `if` here. A bit more explicit. I use `&&` as a boolean operator and as an expression (specifically) never as a statement modifier.